src/qibocal/web/report.py

```diff
@@ -26,3 +27,24 @@ def create_report(path, actions=None):
 
     with open(os.path.join(path, "index.html"), "w") as file:
         file.write(html)
+
+
+def create_autocalibration_report(path, history):
+    """Creates an HTML report for the data in the given path."""
+
+    filepath = pathlib.Path(__file__)
+    with open(os.path.join(filepath.with_name("static"), "styles.css")) as file:
```

Comment:
```suggestion
	STYLES = WEB_DIR / "static" / "styles.css"
    with open(STYLES) as file:
```
Another top-level constant.

You do not need to use `os.path` if you're working with `pathlib.Path` objects.